content/blog/2023-03-02-ethereum-classic-course-11-ethereum-classic-social-and-physical-layer-security/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Ethereum Classic Course: 11. Ethereum Classic Social and Physical Layer Security"
+date: 2023-03-02
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["Course", "Education"]
+linkImage: ./etc-social-physical-banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/0FPTokHcgAE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+## 1. The Complete Security Model of Ethereum Classic
+
+![The complete security model of ETC.](./0.png)
+
+In past classes we have explained how Satoshi Nakamoto invented the proof of work (PoW) based consensus mechanism, what are its revolutionary features and benefits of decentralization, and how the mechanism works in detail.
+
+However, Nakamoto Consensus alone is not the complete security model of a blockchain as Ethereum Classic (ETC). Proof of work is the most important safety barrier of the live operating network, but the operators themselves, who constitute what we call the social layer, collectively represent the other critical layer.
+
+In this class we will explain what are the physical and social layers, their security models in more detail, how they work combined, and what are the possible attack types in each layer.
+
+## 2. The Physical Layer vs the Social Layer 
+
+![The Physical Layer and Social Layer.](./1.png)
+
+The physical layer, which is the live operating network, is the most well known and consists of the protocol rules, the software clients that people must run in their machines to participate, cryptography, and the energy PoW uses that protects the blockchain.
+
+The social layer consists of all the node operators (e.g. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, and independent participants), mining pools, and independent miners. It also includes the developers who conduct research and development to advance the protocol and fix bugs when necessary. All these constituents, plus users and investors who own the native cryptocurrency, decide whether to adopt changes through an improvement proposal process and by running the protocol through software clients.
+
+Although the physical layer may be highly secure and indestructible, the truth is that if all the constituents in the network at the social layer were to agree on bad changes or corrupt practices, they could, technically, do it. 
+
+However, this is extremely difficult because blockchain communities are usually very conservative and principled ecosystems and there is usually ample activism that analyzes and filters proposals and potential changes. This is why the combined systems of the social and physical layers are the complete story of how Ethereum Classic is protected.
```

Comment:
Line 37: Usage
```suggestion
However, this is extremely difficult because blockchain communities are usually very conservative and principled ecosystems where there is usually ample activism that analyzes and filters proposals and potential changes. This is why the combined systems of the social and physical layers are the complete story of how Ethereum Classic is protected.
```